src/dosext/dpmi/dnative/sigsegv.c

```diff
@@ -951,6 +958,10 @@ static void fixupsig(int sig)
 	if (sa.sa_handler == SIG_DFL || sa.sa_handler == SIG_IGN)
 		return;
 	sa.sa_flags |= SA_ONSTACK | SA_SIGINFO;
+	if (signative_block_all_sigs())
```

Comment:
I guess this function can now be removed
and the variable that it currently returns, be
directly used.